app/components/createConfig/createConfig.html

```diff
@@ -0,0 +1,85 @@
+<rd-header>
+  <rd-header-title title="Create config"></rd-header-title>
+  <rd-header-content>
+    <a ui-sref="configs">Configs</a> &gt; Add config
+  </rd-header-content>
+</rd-header>
+
+<div class="row">
+  <div class="col-lg-12 col-md-12 col-xs-12">
+    <rd-widget>
+      <rd-widget-body>
+        <form class="form-horizontal">
+          <!-- name-input -->
+          <div class="form-group">
+            <label for="config_name" class="col-sm-1 control-label text-left">Name</label>
+            <div class="col-sm-11">
+              <input type="text" class="form-control" ng-model="formValues.Name" id="config_name" placeholder="e.g. myConfig">
+            </div>
+          </div>
+          <!-- !name-input -->
+          <!-- config-data -->
+          <div class="form-group">
+            <label for="config_data" class="col-sm-1 control-label text-left">Config</label>
+            <div class="col-sm-11">
+              <textarea class="form-control" rows="5" ng-model="formValues.Data"></textarea>
+            </div>
+          </div>
+          <!-- !config-data -->
+          <!-- encode-config -->
+          <div class="form-group">
+            <div class="col-sm-12">
+              <label for="encode_config" class="control-label text-left">
+                Encode config
```

Comment:
Configs cannot be encoded, this is probably a copy/paste from secrets, could you remove it?

Reply:
According to the ConfigCreate API docs, `Data` needs to be a `base64-url-safe-encoded` string.

https://docs.docker.com/engine/api/v1.32/#operation/ConfigCreate

Reply:
Oh you're right, I missed that part... tho UX wise I think that it would be better remove that field and automatically encode the configuration (I don't think there would be many people pasting encoded configs).

Reply:
I agree. I will change it to always encode the config.